src/OpenSSL/crypto.py

```diff
@@ -2428,7 +2448,7 @@ def get_revoked(self) -> Optional[Tuple[_RevokedInternal, ...]]:
             return tuple(results)
         return None
 
-    def add_revoked(self, revoked: Revoked) -> None:
+    def add_revoked(self, revoked: _RevokedInternal) -> None:
```

Comment:
Did this fail mypy or are we avoiding deprecation warnings?

Reply:
It's for the docs, see the current [type signature for this function](https://www.pyopenssl.org/en/latest/api/crypto.html#OpenSSL.crypto.CRL.add_revoked):
```
add_revoked(revoked: <cryptography.utils._DeprecatedValue object at 0x7f66c71ab1d0>) → None
Add a revoked (by value not reference) to the CRL structure
```